### ISSUE TITLE
Allow disabling blocks of files

### ISSUE THREAD

**Issue by andrewvaughan** [editorconfig-checker/editorconfig-checker#124]
It is extremely common for Markdown files to have link references at the bottom, some of which go well-beyond line maximums.  Markdown syntax does not allow a "comment" to put `<!-- editorconfig-checker-disable-line -->` on these lines.

It would be helpful to be able to disable the checker for blocks of text, as an example (and recommendation):

```md
# My File

...text that is checked by `editorconfig-checker`...

<!-- editorconfig-checker-disable -->

[somelink]:  http://www.some.super.long.url.well.larger.than.the.maximum.line.length/that-I-have-no-control-over.html

<!-- editorconfig-checker-enable -->
```

Similar for other languages, like Python:

```python
import a from b

# editorconfig-checker-disable
some.line("out of compliance")
# editorconfig-checker-enable
```

**Comment by mstruebing** [comment]
Hey @andrewvaughan, great feature request, I agree that this is something which would be really cool, unfortunatly currently it is only possible to exclude single lines or whole files, not blocks.

See: 
https://github.com/editorconfig-checker/editorconfig-checker#excluding-lines
https://github.com/editorconfig-checker/editorconfig-checker#inline

If you want, you could provide a PR (or everyone else) I would be really happy about it :+1:

**Comment by mstruebing** [comment]
A good starting point could be to remove everything between these mentions in this string slice here https://github.com/editorconfig-checker/editorconfig-checker/blob/master/pkg/validation/validation.go#L21
